test/extended/testdata/testjob-plugin.xml

```diff
@@ -27,6 +27,22 @@
       <authToken></authToken>
       <followLog>true</followLog>
       <checkForTriggeredDeployments>true</checkForTriggeredDeployments>
+
```

Comment:
In conjunction with the other comment, create a new test job xml file, that just does a build and includes your new env vars